site/src/github.rs

```diff
@@ -675,8 +690,8 @@ async fn categorize_benchmark(
     if primary.is_none() && secondary.is_none() {
         return (
             format!(
-                "This benchmark run did not return any relevant results.\n\n{}",
-                DISAGREEMENT
+                "This benchmark run did not return any relevant results.\n\n{}{}",
+                DISAGREEMENT, errors
```

Comment:
Maybe we can combine `DISAGREEMENT` + `errors` into some kind of `footer` variable, since it's used in two places? So that we don't forget to update it on both places if something in the footer changes.